From 08daf62c102d81ac67da330d0c40d92b9cbb53ac Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Fri, 13 Dec 2024 15:14:17 +0100
Subject: [PATCH] gettext-full: add missing compilation for language and
 missing dependency

Add missing language compilation and add libiconv dependency for libintl-full
package
---
 package/libs/gettext-full/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/libs/gettext-full/Makefile b/package/libs/gettext-full/Makefile
index e806e477a0..c77ac12695 100644
--- a/package/libs/gettext-full/Makefile
+++ b/package/libs/gettext-full/Makefile
@@ -61,10 +61,12 @@ PKG_MAKEFILES_FAKE:= \
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/libintl-full
   SECTION:=libs
   CATEGORY:=Libraries
+  DEPENDS:=$(ICONV_DEPENDS)
   TITLE:=GNU Internationalization library
   URL:=http://www.gnu.org/software/gettext/
   ABI_VERSION:=8
-- 
2.47.1

